coordinator/meshapi.go

```diff
@@ -49,7 +49,7 @@ func newMeshAPIServer(meshAuth *authority.Authority, bundleGetter certBundleGett
 
 	attestationFailuresCounter := promauto.With(reg).NewCounter(prometheus.CounterOpts{
 		Subsystem: "contrast_meshapi",
-		Name:      "attestation_failures",
+		Name:      "attestation_failures_total",
```

Comment:
Have you considered creating a constant containing this value? We could reuse the constant in the tests.

Reply:
Might be a good idea if we want to use this in more places in the future, or must reference more label names elsewhere. But I think in general building up tests on metrics as insight isn't that good of a design.